src/Symfony/Component/ErrorHandler/ErrorRenderer/ErrorRenderer.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ErrorHandler\ErrorRenderer;
+
+use Symfony\Component\ErrorHandler\Exception\ErrorRendererNotFoundException;
+use Symfony\Component\ErrorHandler\Exception\FlattenException;
+
+/**
+ * Renders an Exception that represents a Response content.
+ *
+ * @see ErrorRendererInterface
+ *
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class ErrorRenderer
+{
+    private $renderers = [];
+
+    /**
+     * @param ErrorRendererInterface[] $renderers
+     */
+    public function __construct(array $renderers)
+    {
+        foreach ($renderers as $renderer) {
+            if (!$renderer instanceof ErrorRendererInterface) {
+                throw new \InvalidArgumentException(sprintf('Error renderer "%s" must implement "%s".', \get_class($renderer), ErrorRendererInterface::class));
```

Comment:
this seems irrelevant because the type is validated using the typehint in `addRenderer`